virtualbox/create-env.sh

```diff
@@ -97,6 +97,14 @@ bosh -n update-cloud-config "${bosh_deployment}/warden/cloud-config.yml" \
 
 echo Succeeded
 
+####
+STEP "Updating DNS Runtime Config"
+####
+
+bosh -n update-config --type runtime --name dns "${bosh_deployment}/runtime-configs/dns.yml" \
```

Comment:
this should probably use `bosh update-runtime-config` instead of the generic `bosh update-config --type runtime`, such that the remote release gets uploaded by the CLI: https://github.com/cloudfoundry/bosh-deployment/blob/master/runtime-configs/dns.yml#L4